code/src/components/NewThoughts.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import Loading from './Loading'
+import CharCounter from './CharCounter'
+
+const NewThought = (props) => {
+  const {onFormSubmit, newThought, setNewThought, loading} = props
+
+  const [countChar, setCountChar] = useState(0)
+
+  return (
+    <div className="container">
+      <form className="inner-container-form" onSubmit={onFormSubmit}>
+        <label htmlFor="newThought">What's making you happy right now?</label>
+        <textarea 
```

Comment:
I liked the textarea approach but as we discussed in the team, it's a bit annoying that you can't press enter to submit. But it's the default I guess.